masonry/src/core/mod.rs

```diff
@@ -34,14 +34,12 @@ pub use widget_pod::WidgetPod;
 pub use widget_ref::WidgetRef;
 pub use widget_state::WidgetOptions;
 
-pub use ui_events::{ScrollDelta, keyboard, pointer};
-
-pub use keyboard::{KeyboardEvent, Modifiers};
 pub use pointer::{
     PointerButton, PointerEvent, PointerId, PointerInfo, PointerState, PointerType, PointerUpdate,
 };
+pub use ui_events::keyboard::{KeyboardEvent, Modifiers};
+pub use ui_events::{ScrollDelta, keyboard, pointer};
```

Comment:
This leaves the line above these `pub use pointer::` still referencing the import of `pub use ui_events::{..., pointer}`; so might as well change it the way you did for keyboard.